src/pages/RepoPage/ComponentsTab/subroute/ComponentsTable/ComponentsTable.tsx

```diff
@@ -125,7 +169,7 @@ function createTableData({
         />
       ),
       lastUploaded: (
```

Comment:
align the Last Uploaded column to the left (because the values of this column are strings)